<launch>

  <!--  Arm Controllers -->
  <rosparam command="load" file="$(find tams_pr2_mannequin_mode)/config/pr2_arm_controllers_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_arm_controllers"
        args="--stopped r_arm_controller_loose l_arm_controller_loose" />

  <!--  Head Controllers -->
  <rosparam command="load" file="$(find tams_pr2_mannequin_mode)/config/head_position_controller_loose.yaml" />
  <node pkg="pr2_controller_manager" type="spawner" name="spawn_head_controller"
        args="--stopped head_traj_controller_loose" />

</launch>
